lms/templates/help_modal.html

```diff
@@ -18,207 +18,191 @@
   <a href="#help-modal" rel="leanModal" role="button">${_("Support")}</a>
 </div>
 
-<section id="help-modal" class="modal" aria-hidden="true" role="dialog" aria-label="${_("{platform_name} Help").format(platform_name=static.get_platform_name())}">
-  <div class="inner-wrapper" id="help_wrapper">
+<section id="help-modal" class="modal" aria-hidden="true" role="dialog" tabindex="-1" aria-label="${_("{platform_name} Help").format(platform_name=static.get_platform_name())}">
```

Comment:
Now that this tab has been renamed `Support`  can we use this opportunity to update the title of this section to be `"{platform_name} Support"` as well